external/escodegen

```diff
@@ -1 +1 @@
-Subproject commit ccf0b688b9d60ef4e25c3da961f5fe40073d1924
+Subproject commit 4c3e29bf1ea41f4b0c0d6ccd21b0b0f43f45f6c6
```

Comment:
It's weird that there were issues with this extern because ```ccf0b688b9d60ef4e25c3da961f5fe40073d1924``` is a legit commit (Merge pull request #1 from kevinb7/yield_with_parentheses).